docs/reference/ccr/apis/get-ccr-stats.asciidoc

```diff
@@ -80,6 +80,12 @@ This object consists of the following fields:
   to the details of the response in the
   <<ccr-get-follow-stats,get follower stats API>>
 
+==== Authorization
+
+If the {es} {security-features} are enabled, you must have `read` and `monitor`
```

Comment:
Cluster monitor privileges are needed to use this api in the follower cluster.

Reply:
I'm assuming this is instead of read and monitor index privileges.  If that's not correct, please let me know.